openapi/components/schemas/Product.yaml

```
@@ -12,6 +12,22 @@ allOf:
         description: Accounting code of the product.
         type: string
         example: '4010'
+      recognition:
+        type: object
```

```suggestion
        type: object
        nullable: true
```
How about making the object nullable?